.github/workflows/release.yaml

```diff
@@ -27,6 +18,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - uses: actions-rust-lang/setup-rust-toolchain@v1.9
+        with:
+          target: x86_64-pc-windows-gnu
```

Comment:
Why windows?

Reply:
Because of shameless copy and paste. Once we include linux in our releases, we anyway won't need to build the scheduler in this step again just to get the version. We will simply use the binary built for linux instead.